apps/examples/App.tsx

```diff
@@ -143,17 +152,21 @@ const MainScreen = ({ navigation }: MainScreenProps): React.JSX.Element => (
           onPress={() => navigation.navigate(name)}
         />
       ))}
-    <Text style={styles.label}>Playgrounds</Text>
-    {Object.keys(SCREENS)
-      .filter(name => SCREENS[name].type === 'playground')
-      .map(name => (
-        <ListItem
-          key={name}
-          testID={`root-screen-playground-${name}`}
-          title={SCREENS[name].title}
-          onPress={() => navigation.navigate(name)}
-        />
-      ))}
+    {!Platform.isTV && (
```

Comment:
Is this if necessary? Maybe some playgrounds will be available for TVOS in the future. If you don't want to show the list if it's empty then we could check length.

Reply:
https://github.com/software-mansion/react-native-screens/pull/2173/commits/4d12007c5b5c5c0ab84dace099e888d7f320c545

Reply:
https://github.com/software-mansion/react-native-screens/pull/2173/commits/4d12007c5b5c5c0ab84dace099e888d7f320c545